lms/templates/discussion/_discussion_inline.html

```diff
@@ -1,5 +1,8 @@
 <%page expression_filter="h"/>
+
+<%include file="/mathjax_include.html" args="disable_fast_preview=disable_fast_preview"/>
 <%include file="_underscore_templates.html" />
+
```

Comment:
@e-kolpakov I think I'll leave that in, the file as whole looks a little bit better with that extra bit of vertical whitespace (to me) :)